tools/build_or_test.sh

```diff
@@ -35,20 +33,5 @@ if [[ $1 != "build" && $1 != "test" ]]; then
   exit 1
 fi
 
-if [[ ! -z "$CIRCLECI" ]]; then
-  echo "$RED""Due to flakiness in bazel execution on CircleCI, trying to build several times"
-  for i in $(seq 1 ${CIRCLECI_MAX_ATTEMPTS}); do
-    echo "$RED""[Attempt $i/${CIRCLECI_MAX_ATTEMPTS}]: building$RESET"
-    bazel_build $1
-    if [[ $? -eq 0 ]]; then
-      echo "$GREEN""[Attempt $i/${CIRCLECI_MAX_ATTEMPTS}]: successful$RESET"
-      exit 0
-    fi
-  done
-
-  echo "$RED""[Attempts exhausted]: Seems it's a problem with your code and not a CircleCI flake.$RESET"
```

Comment:
How come we don't need this anymore?

Reply:
@oferb problems we are having with `bazel` is that in Docker container memory ceiling is not calculated correctly (see the issue I linked in description and [this comment](https://github.com/bazelbuild/bazel/issues/3645#issuecomment-331672268) specifically). 
That way, we re-ran `bazel` until it finally succeeded in hope that it would consume less memory with each run (due to caching). It is no longer needed because we'll be running on a real virtual machine instead of Docker — which means `bazel` is able to calculate memory ceiling correctly and not allocate more than available.

Reply:
Cool :)